dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/utils/JweHelper.java

```diff
@@ -72,6 +98,12 @@ Key getKeyManagementKeyFromTrustStore(JwtConfig jwtConfig) throws KeyStoreExcept
         return JwtUtils.getPublicKey(keyAlias, trustStoreRef);
     }
 
+    Key getJweDecryptionKey(JwtConsumerConfig config) throws KeyStoreException, CertificateException {
```

Comment:
Please confirm if the key value is trace, if so, then tag this method and elsewhere this key is visible in this path with "@Sensitive"